packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -176,3 +270,14 @@ export default class BranchPublish extends Command {
     }
   }
 }
+
+function formatUpdateTitle(
+  update: Pick<Update, 'message' | 'createdAt' | 'runtimeVersion'> & {
+    actor?: Maybe<Pick<User, 'firstName' | 'id'> | Pick<Robot, 'firstName' | 'id'>>;
```

Comment:
```suggestion
    actor?: { firstName: string | null };
```
This type covers all the fields used by this function.